sklearn/inspection/_partial_dependence.py

```diff
@@ -381,6 +391,10 @@ def partial_dependence(
     PartialDependenceDisplay.from_estimator : Plot Partial Dependence.
     PartialDependenceDisplay : Partial Dependence visualization.
 
+    Notes
+    -----
+    NaNs values in the `X` matrix are ignored when computing the partial dependence.
```

Comment:
```suggestion
    NaN values in `X` are ignored when computing the partial dependence.
```
I don't think we need the `s` on NaN. IMHO "NaN values" is "one word".

What do you think about replacing the "the X matrix" with just "X"?